proto/internal.proto

```diff
@@ -324,3 +324,12 @@ message RaftTruncatedState {
   // The term corresponding to 'index'.
   optional uint64 term = 2 [(gogoproto.nullable) = false];
 }
+
+// RaftSnapshotData is the payload of a raftpb.Snapshot. It contains a raw copy of
+// all of the range's data and metadata, including the raft log, response cache, etc.
+message RaftSnapshotData {
+  repeated group KV = 1 [(gogoproto.customname) = "KV"]{
```

Comment:
Because my brain still thinks in terms of proto v1. I'll change it to a nested message.